lib/src/utils/utilities.dart

```diff
@@ -1,8 +1,19 @@
+import 'dart:io';
 import 'dart:typed_data';
 
+import 'package:ledger_cardano/src/utils/validation_exception.dart';
 import 'package:ledger_flutter/ledger_flutter.dart';
 
 Uint8List useBinaryWriter(Uint8List Function(ByteDataWriter writer) invoker) {
   final writer = ByteDataWriter();
   return invoker(writer);
 }
+
+Uint8List ipStringToBytes(String ipString) {
+  try {
+    InternetAddress address = InternetAddress(ipString);
```

Comment:
Can you please check if you can find some other way to do this without using something from `dart:io`? Maybe finding some other library and copying the code from there?

P.s. dart:io import doesn't work for flutter web